src/GeneralStateTestsFiller/stEWASMTests/selfDestructNewAccountFiller.yml

```diff
@@ -0,0 +1,60 @@
+# selfDestruct new account
+---
+selfDestructNewAccount:
```

Comment:
I think this was already done, but under a different name. The `selfDestructNewAccount` case seems to be covered by the basic interface test?